The default makefile sets CXX to g++ and CC to gcc. This leads to a wrong architecture when cross-compiling.
Fix error:
	ERROR: QA Issue: Architecture did not match (62 to 3) on work/.../packages-split/p7zip-dbg/usr/lib/p7zip/.debug/7z.so [arch]
	ERROR: QA Issue: Architecture did not match (62 to 3) on work/.../packages-split/p7zip-full/usr/lib/p7zip/7z.so [arch]
	ERROR: QA Issue: Architecture did not match (62 to 3) on work/.../packages-split/p7zip/usr/lib/p7zip/7zr [arch]
	ERROR: QA run found fatal errors. Please consider fixing them.

diff --git a/makefile.machine b/makefile.machine
index 93e0af9..feca6dd 100644
--- a/makefile.machine
+++ b/makefile.machine
@@ -10,8 +10,8 @@ ALLFLAGS=${OPTFLAGS} -pipe \
 	-D_7ZIP_LARGE_PAGES \
 	$(LOCAL_FLAGS)
 
-CXX=g++ $(ALLFLAGS) -DENV_HAVE_GCCVISIBILITYPATCH -fvisibility=hidden -fvisibility-inlines-hidden
-CC=gcc $(ALLFLAGS)
+CXX=##CXX## $(ALLFLAGS) -DENV_HAVE_GCCVISIBILITYPATCH -fvisibility=hidden -fvisibility-inlines-hidden
+CC=##CC## $(ALLFLAGS)
 CC_SHARED=-fPIC
 LINK_SHARED=-fPIC -shared
 
